frontend/src/metabase/query_builder/components/notebook/Notebook.tsx

```diff
@@ -86,27 +84,5 @@ const Notebook = ({ className, updateQuestion, ...props }: NotebookProps) => {
   );
 };
 
-function getSourceQuestionId(question: Question) {
-  const query = question.query();
-  const { isNative } = Lib.queryDisplayInfo(query);
-
-  if (!isNative) {
-    const sourceTableId = Lib.sourceTableOrCardId(query);
-
-    if (isVirtualCardId(sourceTableId)) {
-      return getQuestionIdFromVirtualTableId(sourceTableId);
-    }
-  }
-
-  return undefined;
-}
-
 // eslint-disable-next-line import/no-default-export -- deprecated usage
-export default _.compose(
-  Questions.load({
-    id: (state: State, { question }: NotebookProps) =>
-      getSourceQuestionId(question),
-    entityAlias: "sourceQuestion",
```

Comment:
The alias here specifies that `sourceQuestion` is passed into the component, but we don't use it in the components so there's no point in having this extra call.